### PR TITLE
Enable publish-knative monitoring and event-service

### PR DESCRIPTION
**Pull request by montaro**
See also #2601

### CODE REVIEW

File: resources/event-bus/charts/publish-knative/templates/_helpers.tpl [hunk 1]
```diff
@@ -1,15 +1,11 @@
 {{- define "publish-knative.name" -}}
-{{- printf "publish-knative" -}}
+{{- printf "%s-%s" .Release.Name "publish" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{- define "publish-knative.fullname" -}}
```
Comment (marcobebway) [inline resources/event-bus/charts/publish-knative/templates/_helpers.tpl]: Do we still need the **publish-knative.fullname**?

Comment (montaro) [reply resources/event-bus/charts/publish-knative/templates/_helpers.tpl]: No, will remove it

File: components/application-operator/charts/application/templates/deployment.yaml [hunk 2]
```diff
@@ -73,6 +73,8 @@ spec:
         sidecar.istio.io/inject: "true"
       labels:
         app: {{ .Release.Name }}-event-service
+        kyma-grafana: enabled
+        kyma-alerts: enabled
```
Comment (Szymongib) [inline components/application-operator/charts/application/templates/deployment.yaml]: This changes will take effect only after you bump image of `application-operator`. You might want to extract the to separate PR and bump image of operator on this one.

Comment (janmedrek) [reply components/application-operator/charts/application/templates/deployment.yaml]: You are setting those to always be `enabled`, what would happen when we use this on the cluster that has no monitoring/alerting? Parametrisation is a good practice and should be applied here.

Please use the `values.yaml` file to parametrise those, there is already `eventService.deployment` field there, you could do something like:
```
labels:
  app: ...
  kyma-grafana: {{ .Values.eventService.deployment.labels.grafana }}
  kyma-alerts: {{ .Values.eventService.deployment.labels.alerts }}
```

and then in `values.yaml`:
```
eventService:
  deployment:
    ...
    labels:
      grafana: "enabled"
      alerts: "enabled"
    ...
```

Comment (montaro) [reply components/application-operator/charts/application/templates/deployment.yaml]: I'll update the `application-operator version` in a following PR after this PR gets merged/image built
